rfc6962-bis.xml

```diff
@@ -375,13 +375,13 @@ REDACT(label) = prefix || BASE32(index || LABELHASH(keyid || label))</artwork>
             <spanx style="verb">prefix</spanx> is the "?" character (ASCII value 63).
           </t>
           <t>
-            <spanx style="verb">||</spanx> denotes concatenation.
+            <spanx style="verb">||</spanx> denotes length-encoded concatenation. Each concatenated value is preceded by a single byte that contains the length (in bytes) of that value.
```

Comment:
"one or more non-wildcard labels **in each DNS-ID** [RFC6125] can each be replaced with a redacted label as follows"